testutil/btcstaking-helper/keeper.go

```diff
@@ -168,8 +193,17 @@ func NewHelperWithStoreAndIncentive(
 	btccKForBtcStaking *types.MockBtcCheckpointKeeper,
 	btccKForFinality *ftypes.MockCheckpointingKeeper,
 	ictvKeeper IctvKeeperI,
+	storeKeys ...*storetypes.KVStoreKey,
 ) *Helper {
-	k, _ := keepertest.BTCStakingKeeperWithStore(t, db, stateStore, nil, btclcKeeper, btccKForBtcStaking, ictvKeeper)
+	var btcStkStoreKey *storetypes.KVStoreKey
+	for _, k := range storeKeys {
```

Comment:
not sure if this is a good choice, I would prefer to update all the calls to call as `nil` in the ones that don't need storeKey and expect to receive the btc staking store key as parameter

but mostly your call on this one